core/src/main/java/bio/terra/pearl/core/service/participant/EnrolleeRelationService.java

```diff
@@ -51,6 +42,19 @@ public Enrollee newGovernedEnrolleeCreationRecord(Enrollee enrollee, Participant
                 .build();
 
         this.create(enrolleeRelation);
-        return  enrollee;
+        return enrollee;
+    }
+
+    @Override
+    @Transactional
+    public void delete(UUID enrolleeRelationId, Set<CascadeProperty> cascades) {
+        dao.delete(enrolleeRelationId);
```

Comment:
I feel like there should be some checks or guards here, but I'm not sure what is appropriate.

Reply:
that's a great question -- let's ask the team.  I suspect the right answer is to make EnrolleeRelationService extend `DataAuditedService` so that we can allow deletes, but we'll always have a record of the changes.  And then let's talk tomorrow about what withdrawal looks like for proxies